app/db/test.py

```diff
@@ -0,0 +1,42 @@
+lines = ['diff --git a/app/main.py b/app/main.py', 'index 09ca0ff..7064c3e 100644', '+++ b/app/main.py', '@@ -6,6 +6,7 @@ from app.api import api', ' ', ' ++++app = FastAPI()', ' ', '+security_token = "qwertyuiopasdfghjklzxcvbnm" ', ' ', ' @app.get("/")', ' def root():', '']
+trimmed_lines = [line.strip().lstrip('+') for line in lines]
+print(trimmed_lines)
+print(trimmed_lines.index('app = FastAPI()'))
```

Comment:
Finding the index of a specific string ('app = FastAPI()') in a list could potentially raise a ValueError if the string is not found. It's safer to handle such cases explicitly to avoid runtime errors.

**Label**: bug

**Code suggestion**

```python
try:
    print(trimmed_lines.index('app = FastAPI()'))
except ValueError:
    print('String not found')
```